.github/workflows/locize-pull-published-sync-pr.yml

```
@@ -0,0 +1,50 @@
+name: Create Translation PR on Version Published
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)